packages/front-end/components/Layout/Layout.tsx

```diff
@@ -45,17 +44,44 @@ const navlinks: SidebarLinkProps[] = [
     path: /^(features)/,
   },
   {
-    name: "Experiments",
+    name: "Experimentation",
     href: "/experiments",
-    path: /^experiment/,
+    path: /^(experiments$|experiment\/|bandit|namespaces)/,
```

Comment:
Does this match the `experiments/templates` route?  Also, power-calculator is missing.

Reply:
fixed